common/script/content/index.js

```diff
@@ -1251,7 +1251,7 @@ api.quests = {
       str: 1.5
     },
     drop: {
-      items: [
+      itemsForOwner: [
```

Comment:
why this name change? It's probably a breaking change that we cannot make without breaking mobile apps